src/organizations/components/DeleteOrgReasonsForm.tsx

```diff
@@ -31,17 +17,25 @@ enum VariableItems {
   OTHER_REASON = 'Other reason',
 }
 
-function DeleteOrgReasonsForm({errors = {}}: ComponentProps) {
-  const [workingVariable, setWorkingVariable] = useState('')
+function DeleteOrgReasonsForm() {
+  const [workingVariable, setWorkingVariable] = useState('USE_CASE_DIFFERENT')
```

Comment:
nit: we have an enum, let's use that:

`useState<VariableItems>(VariableItems.USE_CASE_DIFFERENT)`

Reply:
> This isn't going to give me the key. Instead, it will give me the value of that enum

Reply:
Not sure what that means